src/core/friendly_errors/fes_core.js

```diff
@@ -242,16 +170,17 @@ if (typeof IS_MINIFIED !== 'undefined') {
    *
    * @method _friendlyError
    * @private
-   * @param  {Number} message message to be printed
-   * @param  {String} [method] name of method
-   * @param  {Number|String} [color]   CSS color string or error type
+   * @param  {String}         message   Message to be printed
+   * @param  {String}         [func]    Name of the function linked to error
+   * @param  {Number|String}  [color]   CSS color string or error type
    */
-  p5._friendlyError = function(message, method, color) {
-    p5._report(message, method, color);
+  p5._friendlyError = function(message, func, color) {
```

Comment:
I am wondering if the Contributor docs are generated/updated from this automatically? if not, might be a good work item to add to the "future work" section of the markdown file